src/common/const.go

```diff
@@ -29,58 +29,59 @@ const (
 	RoleDeveloper    = 2
 	RoleGuest        = 3
 
-	ExtEndpoint                = "ext_endpoint"
-	AUTHMode                   = "auth_mode"
-	DatabaseType               = "database_type"
-	MySQLHost                  = "mysql_host"
-	MySQLPort                  = "mysql_port"
```

Comment:
what's the change in these lines? Spaces to Tab?

Reply:
The IDE does some format works to keep the alignment of the variables.